app/views/job_openings/index.slim

```diff
@@ -3,8 +3,10 @@
     .row
       .col-md-4
         label My Name
-        h2= @user.name
+        h3= @user.name
       .col-md-8
+        -if @user.phone_numbers.empty?
```

Comment:
Since you're using Draper, you have access to Rails content helpers via the `h` method.  So, it would look something like:

```ruby
def no_phone_numbers
  h.content_tag(:h4, "Please enter contact information") if object.phone_numbers.empty?
end
```